code/game/gamemodes/clock_cult/clock_items/clock_components.dm

```diff
@@ -11,8 +11,14 @@
 
 /obj/item/clockwork/component/pickup(mob/living/user)
 	..()
-	if(iscultist(user) || (user.mind && user.mind.assigned_role == "Chaplain"))
+	if(iscultist(user) || (user.mind && user.mind.isholy))
 		user << "<span class='[message_span]'>[cultist_message]</span>"
+		if(user.mind && user.mind.isholy)
+			user << "<span class='boldannounce'>[SSreligion.Bible_deity_name]'s voice thunders: BEGONE FALSE THING!</span>"
```

Comment:
also admins force voices on the named and fluffed character every time they subtle voice someone